web/modules/custom/hoeringsportal_project_timeline/assets/js/project-timeline-backend.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * @file
+ * Modify project timeline backend form.
+ */
+
+// Move help text of timeline paragraph.
+let helpText = document.getElementById('edit-field-timeline-items--description')
+let timelineWrapper = document.getElementById('edit-field-project-start-wrapper')
+let helpTextWrapper = document.createElement('p')
```

Comment:
Should this be a `div` element in stead (e.g. if `#edit-field-timeline-items--description` contains block content)?

Reply:
Yeah probably. Fixed.